rspec-mode.el

```diff
@@ -225,6 +225,13 @@ info, are considered errors."
   :type 'boolean
   :group 'rspec-mode)
 
+(defcustom rspec-holder-dirs '("app" "lib")
```

Comment:
The problem with both holder and container is that they don't convey what sort of thing is held to the reader. Perhaps `rspec-source-dirs` since @dgutov described these as "directories that contain source files"?